src/styles/blocks/page.scss

```diff
@@ -0,0 +1,37 @@
+.page {
+  font-family: "Manrope", sans-serif;
+  font-size: 16px;
+  line-height: 140%;
+  color: $main-text-color;
+
+  scroll-behavior: smooth;
+
+  &:has(.page__menu:target) {
+    overflow: hidden;
+  }
+
+
+  &__body {
+    margin: 0;
+    min-width: 320px;
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+
+    opacity: 0;
+    transition: all 0.3s;
```

Comment:
For `transition`, it is also worth specifying which properties to use it for.

Reply:
Please make sure to fix the comment